src/logger/MeetingSessionPOSTLogger.ts

```diff
@@ -27,7 +27,8 @@ export default class MeetingSessionPOSTLogger implements Destroyable {
     private batchSize: number,
     private intervalMs: number,
     private url: string,
-    private level = LogLevel.WARN
+    private level = LogLevel.WARN,
+    private headers?: Record<string, string>
```

Comment:
In TypeScript DOM types, `headers` is of the `HeadersInit` type.
https://github.com/microsoft/TypeScript/blob/c671fe1bc2878e1ba60f4199856d1a57c3cc99ff/lib/lib.dom.d.ts#L1618

```
type HeadersInit = Headers | string[][] | Record<string, string>;
```

Reply:
Discussed with Kyu offline, based on the existing pattern in the JS SDK https://github.com/aws/amazon-chime-sdk-js/blob/6596622238e69c5202c3e697aeeedd54f2a0d35b/libs/voicefocus/types.d.ts#L125 we will follow only the Record<string, string> type as of now.